src/plone/restapi/tests/test_services_types.py

```diff
@@ -1,15 +1,19 @@
 # -*- coding: utf-8 -*-
+from pkg_resources import parse_version
 from plone import api
-from plone.restapi.testing import PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
-from plone.restapi.testing import RelativeSession
 from plone.app.testing import setRoles
-from plone.app.testing import TEST_USER_ID
 from plone.app.testing import SITE_OWNER_NAME
 from plone.app.testing import SITE_OWNER_PASSWORD
+from plone.app.testing import TEST_USER_ID
+from plone.restapi.testing import PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+from plone.restapi.testing import RelativeSession
+
 
 import transaction
 import unittest
 
+PLONE_VERSION = parse_version(api.env.plone_version())
```

Comment:
This is unused I think.

Reply:
Yeah, traces of the code that I had before your changes :)